test/unit/ObjectMakrTests/make.js

```diff
@@ -0,0 +1,38 @@
+define(["mocks"], function (mocks) {
+    return function () {
+        var expect = require("chai").expect;
+
+        it("copies a string", function () {
+            // Arrange
+            var objectMaker = mocks.mockObjectMakr();
+
+            // Act
+            var madeObject = objectMaker.make(mocks.mockClassName, mocks.mockObjectProperties());
+
+            // Assert
+            expect(typeof madeObject.name).to.equal("string");
+        });
+
+        it("implements an onMake function", function () {
+            // Arrange
+            var objectMaker = mocks.mockObjectMakr();
+
+            // Act
+            var madeObject = objectMaker.make(mocks.mockClassName, mocks.mockObjectProperties());
+
+            // Assert
+            expect(madeObject.name).to.equal("John the Penguin");
```

Comment:
I really don't like `"the Penguin"`. You're relying on test code in another file that isn't mentioned here.

Instead, how about passing an `"onMake"` function to the mocked ObjectMakr that sets a locally scoped variable to true?